source/facter/3.0/release_notes.md

```diff
@@ -29,22 +126,22 @@ Shipped in puppet-agent version: 1.2.0.
 
-For many workflows, this rewrite is a drop-in replacement for the Ruby-based Facter 2.x. It still supports custom facts written in Ruby with the existing Facter API, as well as external facts written in any number of languages.
+For many workflows, this rewrite is a drop-in replacement for the Ruby-based Facter 2. It still supports custom facts written in Ruby with the existing Facter API, as well as external facts written in any number of languages.
 
-It does include a few breaking changes relative to Facter 2.4.
+It includes a few breaking changes relative to Facter 2.4.
 
 * [All tickets fixed in 3.0.0](https://tickets.puppetlabs.com/issues/?filter=14556)
 * [Issues introduced in 3.0.0](https://tickets.puppetlabs.com/issues/?filter=14557)
 
-### BREAK: Facter Doesn't Display Legacy Unstructured Facts with Structured Equivalents
+### BREAK (workaround in 3.0.2): Facter Doesn't Display Legacy Unstructured Facts with Structured Equivalents
 
-Facter 2 introduced structured facts, and Facter 3.0.0 reprovisions many unstructured Puppet facts with new structured equivalents. Facter still tracks the legacy unstructured facts, and `puppet facts` still outputs these facts, but in Facter 3.0.0 these legacy facts no longer appear in the default command-line output. This might break workflows that rely on legacy facts appearing in Facter output.
+Facter 2 introduced structured facts, and Facter 3 reprovisions many unstructured Puppet facts with new structured equivalents. For example, the new map-structured `os` fact describes several legacy operating system-related facts, such as `architecture` and `operatingsystem`.
 
-For example, the new map-structured `os` fact describes several legacy operating system-related facts, such as `architecture` and `operatingsystem`.
+Facter still tracks the legacy unstructured facts, and `puppet facts` still outputs these facts, but in Facter 3 these legacy facts no longer appear in the default command-line output. This change can break workflows that rely on legacy facts appearing in Facter output.
 
 Facts that no longer appear in command-line output are documented as such in [the list of core facts](./core_facts.html).
 
-To display legacy facts on the command line with Facter 3, we recommend either using `puppet facts` or modifying Facter workflows to instead use the equivalent structured facts. A future version of Facter will provide a flag that forces Facter to output deprecated legacy facts as an interim solution for affected workflows.
+To display legacy facts on the command line with Facter 3, we recommend either using `puppet facts` or modifying Facter workflows to instead use the equivalent structured facts. Facter 3.0.2 provides a flag that forces Facter to output deprecated legacy facts as an interim solution for affected workflows.
```

Comment:
I'm not sure if we've decided to call the legacy facts "deprecated" yet.  Facter hides them from its default output to reduce redundancy where the "flat" fact values have been duplicated into better structured facts.  Because there are so many manifests out there that use what we're considering "legacy" facts, I don't think we have any plans in place to ever remove them from Facter entirely.